src_assets/common/assets/web/config.html

```diff
@@ -1159,6 +1173,7 @@ <h1 class="my-4">Configuration</h1>
           let resolutions = [];
           res.split(",").forEach((r) => resolutions.push(r.trim()));
           this.resolutions = resolutions;
+          this.max_bitrate = this.config.max_bitrate || 0;
```

Comment:
We don't need this line for a simple int or str value. We just do it for more complex, json encoded values.